branchff

```diff
@@ -55,6 +55,18 @@ PROG=${0##*/}
 usage=${1:-yes}
 
 source $(dirname $(readlink -ne $BASH_SOURCE))/lib/common.sh
+
+# Check prerequisites
+
+PREREQUISITE_TOOLS=("git" "jq" "go" "make")
+for PREREQ in ${PREREQUISITE_TOOLS[@]}; do
+  command -v $PREREQ > /dev/null 2>&1 || common::exit 1 \
+                                         "$PREREQ not installed"
```

Comment:
I think these two lines can be merged into one

Reply:
Thank you so so much for looking int this. I put the line break so that it fits into the ####### comments within the code. Let me know if you'd like me to make them one line but over the ###### delimiters.

Reply:
it's a good consideration! I think it's not that strict since other lines are preferred in one line too.

Reply:
Sure sounds good. I will make this change. Thanks again for your very helpful feedback.

Reply:
I made the changes and merged to my master branch :)